servers/testkube/server.yaml

```
@@ -0,0 +1,49 @@
+name: testkube
+image: mcp/testkube
+type: server
+meta:
+  category: devops
+  tags:
+    - devops
+    - testing
+    - automation
+    - orchestration
+    - kubernetes
+    - gitops
+    - playwright
+    - cypress
+    - selenium
+    - k6
+    - postman
+    - jmeter
+
+about:
+  title: Testkube
+  description: "The Testkube MCP Server exposes continuous testing capabilities (test orchestration, execution, troubleshooting and anlysis) to AI tools and workflows "
+  icon: https://avatars.githubusercontent.com/u/82541796?v=4
+source:
+  project: https://github.com/kubeshop/testkube
+  branch: 6da58c790ecffdec93de52f6bcd671317a7e2f5c
```

Thanks: @niyiomotoso looks like still failing validation should be like:
```suggestion
  branch: main
  commit: 6da58c790ecffdec93de52f6bcd671317a7e2f5c
```
looks like branch ended up with the hash. We need separate branch and commit ([example](https://github.com/docker/mcp-registry/blob/main/servers/SQLite/server.yaml#L15-L16))